Infrastructure-Support-of-Publication-Process.md

```diff
@@ -0,0 +1,35 @@
+# Infrastructure Support for DevRel Publication Process
+
+This process provides instructions for setting up and updating the [EOS Dev Portal](docs.eosnetwork.com). 
+
+## Build a New Document Repository
+
+The following link provides instructions on how to build a new document repository. Use this procedure to initiate a new document repository for the EOS dev portal (infrastructure), when necessary.  
+
+[How to Build a New Repository](https://github.com/eosnetworkfoundation/devdocs/blob/main/docs/How-To-Do-A-Full-Release.md) 
+
+## How to Automate Documentation 
```

Comment:
The name and description for this is misleading

That document doesn't explain how to automate documentation generation, just how to generate it in an automated fashion

There's a typo: `Autopmate`

Reply:
Done